src/components/MobilePage/MobilePage.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import { Header, Field, Button, Form } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { getAnalytics } from 'decentraland-dapps/dist/modules/analytics/utils'
+
+import { Props, State } from './MobilePage.types'
+
+import './MobilePage.css'
+import { isMobile } from 'decentraland-dapps/dist/lib/utils'
+
+export default class MobilePage extends React.PureComponent<Props, State> {
+  state = {
+    email: ''
+  }
+
+  componentDidMount() {
+    if (!isMobile()) {
+      window.location.href = '/'
+    }
+    document.body.classList.add('mobile')
+  }
+
+  componentWillUnmount() {
+    document.body.classList.remove('mobile')
+  }
+
+  handleEmailChange = (event: React.FormEvent<HTMLInputElement>) => {
+    const email = event.currentTarget.value
+    this.setState({ email })
+  }
+
+  handleSubmit = () => {
+    const { email } = this.state
+    const analytics = getAnalytics()
+    analytics.identify({ email }, () => {
+      analytics.track('Mobile email', { email }, () => {
+        window.location.href = 'https://contest.decentraland.org/'
+      })
+    })
+  }
+
+  render() {
+    const { email } = this.state
+
+    return (
+      <div className="MobilePage">
+        <Header size="large">{t('mobilepage.title')}</Header>
+        <p className="subtitle">{t('mobilepage.subtitle')}</p>
+        <Form onSubmit={this.handleSubmit}>
+          <p className="message">{t('mobilepage.message')}</p>
+          <div className="form-container">
+            <Field type="email" icon="asterisk" placeholder="mail@domain.com" value={email} onChange={this.handleEmailChange} required />
+            <Button primary size="medium">
+              {t('global.send').toUpperCase()}
```

Comment:
this can be done with css: `text-transform: uppercase`

Reply:
yup, will need to change it on the contest too I think (copy pasted)

Reply:
that should be a style of the `<Button>` component tho

Reply:
I just checked `decentraland-ui`, it already does that, so you can just remove the `.toUpperCase()` and it should still show all in caps